osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -49,6 +50,9 @@ public class HUDOverlay : Container, IKeyBindingHandler<GlobalAction>
         public readonly HoldForMenuButton HoldToQuit;
         public readonly PlayerSettingsOverlay PlayerSettingsOverlay;
 
+        [Resolved(canBeNull: true)]
+        private KeysPerSecondCalculator keysPerSecondCalculator { get; set; }
```

Comment:
Oh it's because I thought it caused to fail the discussed test in `SkinDeserializationTest` (see https://github.com/ppy/osu/pull/19599#issuecomment-1212047316 and 787dee24), turns out it's because of the component itself being new. Fixed it, I chose the `HUDOverlay` way in d5f10cb.